ape_safe/accounts.py

```diff
@@ -208,7 +208,63 @@ def prepare_transaction(self, txn: TransactionAPI) -> TransactionAPI:
         # NOTE: Need to override `AccountAPI` behavior for balance checks
         return self.provider.prepare_transaction(txn)
 
-    def call(self, txn: TransactionAPI, **call_kwargs) -> ReceiptAPI:  # type: ignore[override]
+    def _impersonated_call(self, txn: TransactionAPI, **safe_tx_kwargs) -> ReceiptAPI:
+        safe_tx = self.create_safe_tx(txn, **safe_tx_kwargs)
+        safe_tx_exec_args = list(safe_tx._body_["message"].values())
+        signatures = []
+        # Bypass signature collection logic and attempt to submit by impersonation
+        # NOTE: Only works for fork and local networks
+        if self.confirmations_required > 1:
+            safe_tx_hash = self.contract.getTransactionHash(*safe_tx_exec_args)
+            for signer_address in self.signers[: self.confirmations_required - 1]:
+                impersonated_signer = self.account_manager.test_accounts[signer_address]
+                self.contract.approveHash(safe_tx_hash, sender=impersonated_signer)
+                signatures.append(
+                    MessageSignature(
+                        v=1,  # Approved hash (e.g. submitter is approved)
+                        r=b"\x00" * 12 + to_bytes(hexstr=impersonated_signer.address),
+                        s=b"\x00" * 32,
+                    )
+                )
+
+        impersonated_sender = self.account_manager.test_accounts[
+            self.signers[self.confirmations_required - 1]
+        ]
+        signatures.append(
+            MessageSignature(
+                v=1,  # Approved hash (e.g. submitter is approved)
+                r=b"\x00" * 12 + to_bytes(hexstr=impersonated_sender.address),
+                s=b"\x00" * 32,
+            )
+        )
```

Comment:
can be merged with above

Reply:
I don't think so, we are skipping the final submitter in the for loop above, so we need to add them here

Reply:
can just skip sending a tx for them in the loop

Reply:
Actually sorta agree with this one now, just gonna mock this guy out and allow any submitter